README.md

```diff
@@ -51,6 +51,46 @@ This document provides the information needed to create a DASH packager that is
 
   See https://github.com/google/edash-packager/blob/master/CONTRIBUTING.md for details.
 
+# Using docker for testing/development #
+
+> Docker allows you to package an application with all of its dependencies into a standardized unit for software development.
+>
+> -- <cite>Official site</cite>
+
+If you want try or test `edash-packager` you can just use this.
+
+1. [Install Docker.](https://docs.docker.com/installation/)
+
+2. Build the image
+
+```Shell
+docker build -t edash github.com/google/edash-packager.git
+```
```

Comment:
I think you need to shift the text otherwise the list number is not generated correctly. Compare this:

1. item 1
2. item 2
```Sheel
xx
```
3. item3

v.s.

1. item1
2. item2

    ```Shell
    xx
    ```
3. item3